tests/model/operation/transform/insert.js

```diff
@@ -906,74 +933,7 @@ describe( 'transform', () => {
 
 				syncClients();
 
-				expectClients(
-					'<paragraph>BarFo<m1:start></m1:start>o<m1:end></m1:end></paragraph>'
-				);
-			} );
-		} );
-
-		// This should be moved to attribute.js.
-		describe( 'by remove attribute', () => {
```

Comment:
Why you removed those tests?

Reply:
I moved all `remove attribute` tests to the `attribute.js` file.

Reply:
Okay, didn't see that cause attribute.js was not expanded. What about the wrap test?

Reply:
Whoops, I see that wrap, unwrap and split don't have this kind of tests. I'll add them.

Reply:
I've meant that there is a removed test in `move.js`: `'intersecting wrap'`. I've commented on that below.

Reply:
>Whoops, I see that wrap, unwrap and split don't have this kind of tests. I'll add them.

Done.